mm2src/mm2_bitcoin/keys/src/public.rs

```diff
@@ -80,9 +80,11 @@ impl Public {
             Public::Normal(_) => None,
         }
     }
+
+    pub fn to_secp256k1_pubkey(&self) -> Result<PublicKey, SecpError> { PublicKey::from_slice(self.deref()) }
```

Comment:
How frequently is this function called? If it's called very often, it will be better `#[inline(always)]` it.

Reply:
[72d6754](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2408/commits/72d6754e2c0ba1a2c3e55fe0c449c26bd7701b62)

thanks